pysat/instruments/sw_f107.py

```diff
@@ -270,15 +324,18 @@ def download(date_array, tag, sat_id, data_path, user=None, password=None):
             # process
             raw_dict = json.loads(r.text)['noaa_radio_flux']
             data = pds.DataFrame.from_dict(raw_dict['samples'])
-            times = [pysat.datetime.strptime(time, '%Y %m %d')
-                     for time in data.pop('time')]
-            data.index = times
-            # replace fill with NaNs
-            idx, = np.where(data['f107'] == -99999.0)
-            data.iloc[idx,:] = np.nan
-            # create file
-            data.to_csv(os.path.join(data_path, 'f107_monthly_' +
-                                     date.strftime('%Y-%m') + '.txt'))
+            if data.empty:
+                print("WARNING: no data for {:}".format(date))
```

Comment:
Should this be an official warning, like raise RuntimeWarning()

Reply:
If you like, I've not used the official Warnings before.

Reply:
Changed to UserWarning